app/views/devise/shared/_links.html.slim

```diff
@@ -0,0 +1,6 @@
+- if controller_name != 'sessions'
+  = link_to 'Have an account? Log In!', new_session_path(resource_name)
+
+- if devise_mapping.recoverable? && controller_name != 'passwords' && controller_name != 'registrations'
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/nimblehq/nimble-survey-web/blob/b2f39155e54d78bea577b04a53a9496122f58f84/app/views/devise/shared/_links.html.slim#L4"></span>Line is too long. [104/80]</td>
    </tr>
  </tbody>
</table>